third_party/java/java_tools/README.md

```diff
@@ -0,0 +1,19 @@
+# Tools required by the Bazel rules for Java.
+
+To see how to update the tools run:
+
+```
+./third_party/java/java_tools/update_java_tools.sh help
```

Comment:
When do we expect this script to be run? i.e. as part of the release process, or on-demand, or something else?